src/main/java/duke/ReadFile.java

```diff
@@ -0,0 +1,25 @@
+package duke;
+
+import java.util.ArrayList;
+
+//class to read file and process file into a string format
+public class ReadFile {
+    public static String readFile(ArrayList<Task> list) {
+        int n = list.size() - 1;
+        String s = "";
+
+        while (n >= 0) {
```

Comment:
Maybe you can user a for loop to iterate through the list, instead of using a variable named 'n'.

Reply:
I agree with @Jacky142857 here